packages/button/Button.tsx

```diff
@@ -1,33 +1,44 @@
-import { h, Component, prop } from 'skatejs';
+import { h, Component, prop, ComponentProps } from 'skatejs';
 
-import { ColorType, cssClassForColorType, css } from '../_helpers';
+import {
+  ColorType,
+  cssClassForColorType,
+  css,
+  IntrinsicCustomElement,
+  IntrinsicBoreElement,
+  ClickEvent
+} from '../_helpers';
 
 import styles from './Button.scss';
 
-// public
-type ButtonProps = Props & EventProps;
-type Props = {
+export type ButtonProps = Props & Attrs & EventHandlers;
+export type Attrs = {
```

Comment:
yes I got the idea but I'm not sure what does it mean

> only if primary API is via attributes

in reality. How you decide that that particular property is primarily attribute? Also I though that we won't test attributes as that's functionality which is tested in skatejs and in future it will be by default with `attribute:{source: true}`.

Let's discuss this with rest directly wdyt @hopkins-tk @davidrus?